scripts/openvino_accelerate.py

```diff
@@ -595,11 +604,38 @@ def get_diffusers_sd_model(model_config, sampler_name, enable_caching, openvino_
         state_dict = get_checkpoint_state_dict(checkpoint_info, timer)
         checkpoint_config = sd_models_config.find_checkpoint_config(state_dict, checkpoint_info)
         print("OpenVINO Script:  created model from config : " + checkpoint_config)
-        if model_config != "None":
-            local_config_file = os.path.join(curr_dir_path, 'configs', model_config)
-            sd_model = StableDiffusionPipeline.from_single_file(checkpoint_path, local_config_file=local_config_file, load_safety_checker=False)
+        if(is_xl_ckpt):
+
+            if model_config != "None":
+                local_config_file = os.path.join(curr_dir_path, 'configs', model_config)
+                sd_model = StableDiffusionXLPipeline.from_single_file(checkpoint_path, local_config_file=local_config_file, use_safetensors=True)
+            else:
+                sd_model = StableDiffusionXLPipeline.from_single_file(checkpoint_path, local_config_file=checkpoint_config, use_safetensors=True)
+            if (mode == 1):
+                sd_model = StableDiffusionXLImg2ImgPipeline.from_single_file(**sd_model.components)

```

Comment:
@qiacheng, do we need "from_single_file" method when we reuse the components. For example, for the vanilla img2img  pipeline, we just use 'StableDiffusionImg2ImgPipeline(**sd_model.components)' without 'from_single_file'

Reply:
thanks @ynimmaga. code is fixed